src/states/new_hampshire/main.py

```diff
@@ -68,7 +70,20 @@ def extract_city_without_ward(row):
     return capitalized_city.strip()
 
 
+def fetch_data():
+  if os.path.exists(CACHE_FILE_PATH):
+    print('Using cached data')
+  else:
+    process = CrawlerProcess({
+      'FEED_FORMAT': 'jsonlines',
```

Comment:
Instead of using `jsonlines`, use `json` -- I believe you don't have to reformat the data in that case and just output to public directly.